jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -212,8 +213,9 @@
                    viewer_invalid_msg.length > 0 ||
                    (viewer_selected.length > 0 && viewer_format_selected == 'mp4' && !movie_enabled)"
       >
-        Export
+        {{action_label}}{{label_overwrite ? ' (Overwrite)' : ''}}

```

Comment:
Kyle, when you said, "Could just switch the button to say overwrite as we do for data entries, etc," did you mean something like this or am I in the wrong rabbit hole? I think you mentioned that I could copy paste from somewhere but doing it this way might require updating `plugin-action-button` template, so I am probably doing this wrong? Hope you can advise. Thanks!

Reply:
I think the "easiest" way for now (especially given that we're likely going to replace this) is to copy the logic from what exists in the in-plugin exporting (moment maps, collapse, etc), which checks if the file exists after clicking the button, and if so, displays an overlay.


https://github.com/spacetelescope/jdaviz/assets/877591/22f76536-06c9-4353-8ce6-2be3a0e1fdc4